drivers/staging/media/rpivid/rpivid_h265.c

```diff
@@ -20,7 +20,9 @@
 #include "rpivid_hw.h"
 
 #define DEBUG_TRACE_P1_CMD 0
-#define DEBUG_TRACE_EXECUTION 0
+#define DEBUG_TRACE_EXECUTION 1
+
+#define USE_REQUEST_PIN 1
```

Comment:
Is there a need for it to work with ```USE_REQUEST_PIN 0```?

Reply:
Not really but how this bit of the framework works seems somewhat fluid & I was keeping my options open on how to implement it.  At the point I try to upstream any of this there is going to be a massive argument around how this is meant to work (I've seem previews on slack & IRC) unless someone else has forced through framework changes that simplify the situation